itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/txn/compactor/CompactorOnTezTest.java

```diff
@@ -217,6 +224,64 @@ void insertTestData(String dbName, String tblName) throws Exception {
       executeStatementOnDriver("delete from " + tblName + " where a = '1'", driver);
     }
 
+    void createTableWithoutBucketWithMultipleSplits(String dbName, String tblName, String tempTblName,
```

Comment:
Could we leave a comment with the expected file structure?

Reply:
Sure, added a comment.